contrib/node/examples/src/node/web-component-button/BUILD

```diff
@@ -29,3 +29,32 @@ node_bundle(
   name='web-component-button-bundle',
   node_module=':web-component-button'
 )
+
+# The following target invokes webpack build.
+node_module(
+  name='web-component-button-processed',
+  sources=globs('package.json', 'webpack.config.js', 'src/*', 'test/*'),
+  dependencies=[
+    'contrib/node/examples/3rdparty/node/mocha',
+    'contrib/node/examples/3rdparty/node/react',
+    'contrib/node/examples/src/node/web-build-tool',
+  ],
+  build_script='build',
+)
+
+node_bundle(
+  name='web-component-button-processed-bundle',
+  node_module=':web-component-button-processed'
+)
+
+node_module(
```

Comment:
Maybe a comment indicating why both this and the above target exist?

Could this target depend on the other one?